src/libraries/pkg/runtime.native.System.IO.Ports/runtime.native.System.IO.Ports.pkgproj

```diff
@@ -1,4 +1,10 @@
 ﻿<Project DefaultTargets="Build">
+  <PropertyGroup>
+    <BuildingAnOfficialBuildLeg Condition="'$(BuildingAnOfficialBuildLeg)' == '' and '$(OfficialBuildId)' != '' and '$(DotNetBuildFromSource)' != 'true'">true</BuildingAnOfficialBuildLeg>
```

Comment:
@hoyosjs, I had to duplicate this line to use it in the condition in the next line. That appears to have fixed the duplicate System.IO.Ports packages (I was taking local diffs of directory tree on Linux with the command you posted). I have my fingers crossed on the next Official leg run (but first he green CI). :)